tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "allowJs": true,
     "sourceMap": true,
     "moduleResolution": "node",
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "noEmit": true
```

Comment:
Is this because we are using `rollup` to build the modules?

Reply:
Correct [-more info here](https://www.typescriptlang.org/tsconfig/#noEmit)